### ISSUE TITLE
something wrong with the picture from KDE install theme

### ISSUE THREAD

**Issue by dennisTGC** [stepan-zubkov/sddm-zust#12]
It looks like this:
![image](https://github.com/stepanzubkov/sddm-zust/assets/12065502/c8847163-8f16-4652-924d-b5a8ab0dd7bd)

But I think it should look like this:
![image](https://github.com/stepanzubkov/sddm-zust/assets/12065502/f39c36e5-e792-4de5-a968-8f4bf5429b7f)

**Comment by stepanzubkov** [comment]
No, this is theme avatar. Maybe I will replace it by theme screenshot, because avatar was temporary solution. Thanks for issue.
